health-services/project-factory/src/server/utils/campaignUtils.ts

```
@@ -1009,99 +989,97 @@
   ) {
-    enrichRootProjectId(request.body);
+    await enrichRootProjectId(requestBody?.CampaignDetails);
   } else {
-    request.body.CampaignDetails.projectId = null;
+    requestBody.CampaignDetails.projectId = null;
   }
   const topic = firstPersist
     ? config?.kafka?.KAFKA_SAVE_PROJECT_CAMPAIGN_DETAILS_TOPIC
     : config?.kafka?.KAFKA_UPDATE_PROJECT_CAMPAIGN_DETAILS_TOPIC;
-  delete request.body.CampaignDetails.codesTargetMapping;
+  delete requestBody.CampaignDetails.codesTargetMapping;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)